test/cli_exec_test.go

```diff
@@ -242,3 +243,14 @@ func (suite *PouchExecSuite) TestExecWithWorkingDir(c *check.C) {
 		c.Fatalf("failed to run exec with specified working directory: %s, but got %s", dir, res.Stdout())
 	}
 }
+
+// TestExecWithTty tests running container with -tty flag and attach stdin in a non-tty client.
+func (suite *PouchExecSuite) TestExecWithTty(c *check.C) {
+	name := "TestExecWithTty"
+	res := command.PouchRun("run", "-d", "--name", name, busyboxImage, "sleep", "100000")
```

Comment:
it's just nit. could we use one line to check `command.PouchRun("run", "-d", "--name", name, busyboxImage, "sleep", "100000").Assert(c, icmd.Success)` ？ thanks.

Reply:
Done.